docs/DeveloperGuide.md

```diff
@@ -505,38 +516,38 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case ends.
 
 
-**Use case: Importing data from .csv file**
+**Use case: UC 10. Importing data from [CSV](#glossary) file**
 
 **MSS**
 1. User has a data file with resident’s information, and wants to view it in RC4HDB.
-2. User uses the import command to import the CSV file.
-3. RC4HDB reads the file stored in the file path entered by the user.
-4. RC4HDB displays the name of the csv file after the file has been read.
+2. User imports the file.
+3. RC4HDB reads the file.
+4. RC4HDB displays the name of the [**CSV**](#glossary) file after the file has been read.
 5. RC4HDB displays all the information stored in the file.
 
     Use case ends.
 
-**Extension**
-* 1a. Information in the CSV file has not been stored in the proper format.
+**Extensions**
+* 2a. Information in the [**CSV**](#glossary) file has not been stored in the proper format.
 
-    * 1a1. RC4HDB shows an error message.
+    * 2a1. RC4HDB shows an error message.
 
       Use case resumes at step 2.
 
 
-* 2a. No file could be found at the specified file path.
+* 3a. No file could be found at the specified file path.
 
-    * 2a1. RC4HDB shows an error message.
+    * 3a1. RC4HDB shows an error message.
 
       Use case resumes at step 2.
 
 
-**Use case: Exporting data to .csv file**
+**Use case: UC 11. Exporting data to [CSV](#glossary) file**
 
 **MSS**
-1. User wants the data in a CSV file.
-2. User uses the export command, followed by the full file path and new file name.
-3. RC4HDB creates a new CSV file in the file path entered by the user.
+1. User wants the data in a [**CSV**](#glossary) file.
+2. User exports the file.
+3. RC4HDB creates a new [**CSV**](#glossary) file using user input.
```

Comment:
I think maybe we could change this around.
2. User specifies which file they want to export.
3. RC4HDB copies the contents of the file onto a **CSV** file.

Reply:
I don't think you have to update this, we'll probably update this when we implement this feature. Our implementation might change along the way.

Reply:
Same goes for all other UC that have not been implemented yet or are going to be modified.

Reply:
Similar to my comment about the import command, an alternative would be "User wants to save the current data into another CSV file".

Reply:
I'm not going to update this as of now so that we can update it in the future.